tests/Samples.Server.Tests/BaseTest.cs

```diff
@@ -46,23 +39,46 @@ protected async Task<string> SendRequestAsync(string text)
             return await response.Content.ReadAsStringAsync();
         }
 
-        protected async Task<string> SendRequestAsync(GraphQLRequest request)
+        protected async Task<string> SendRequestAsync(GraphQLRequest request, RequestType requestType, string requestUri = "graphql")
```

Comment:
I propose to change `protected async Task<string> SendRequestAsync(string text)` to this:
```c#
protected Task<string> SendRequestAsync(string text) => SendRequestAsync(new GraphQLRequest { Query = text }, RequestType.PostWithJson);
```

Reply:
I'd done this at one point, but the issue is that the test that uses this is specifically passing something that's not even a JSON object at all. Just literally a POST with `Oops` raw body and that's it.

Reply:
Anyway, we specify 'application/json' as mime so the meaning of the test does not change. My interest is to use your new method to sending graphql requests as a single entry point.

Reply:
I can make it even more "raw" and then do a bunch of test cases checking for http method, etc. and all our error statuses. Then this will be more useful.

Reply:
It would be great.